demo/src/main/java/net/minestom/demo/commands/GiveCommand.java

```diff
@@ -51,7 +51,7 @@ public GiveCommand() {
 
             sender.sendMessage(Component.text("Items have been given successfully!"));
 
-        }, Entity("target").onlyPlayers(true), ItemStack("item"), Integer("count").setDefaultValue(() -> 1));
+        }, Entity("target", serverFacade.getInstanceManager(), serverFacade.getConnectionManager()).onlyPlayers(true), ItemStack("item"), Integer("count").setDefaultValue(() -> 1));
```

Comment:
Argument types don't hold any state, so could probably be reused across server instances and then provide the server object via the CommandContext. Need to think a bit more about this but wanted to write it now anyway.